SETUP.md

```diff
@@ -106,6 +108,39 @@ environment. Always use the rails container instead.**
 Any changes to source files should be made directly in your local filesystem under the
 `/rails` directory using your preferred editing tools.
 
+## Backup and restore the Terrastories database
+
+Terrastories stores Places, Speakers, and Stories in a database (Postgres DB). it is possible to back these data up and restore them by running lines of code in a bash terminal.
+
+Backup the DB with:
+
+```
+docker run --rm -v "terrastories_postgres_data:/pgdata" busybox tar -cvzf - -C /pgdata . >db-backup.tgz 
+```
+
+Restore a backup with:
+
+```
+docker volume rm terrastories_postgres_data
+docker run --rm -i -v "terrastories_postgres_data:/pgdata" busybox tar -xvzf - -C /pgdata <db-backup.tgz
+```
+
+**For Windows** applications like PowerShell (PS), a slightly different syntax is needed. 
+
+Backup the DB in PS with:
+
+```
+docker run --rm -v "terrastories_postgres_data:/pgdata" -v "$(pwd):/host" busybox tar -cvzf /host/db-backup-test.tgz -C //pgdata .
```

Comment:
The `//pgdata` syntax should only be needed for Git bash. With PS I believe it will work with `/pgdata`. Pleasantly surprised to hear that `$(pwd)` is working in PS!